{
    "status": "Failed",
    "error": {
        "code": "DeploymentFailed",
        "target": "/subscriptions/0fbef83c-cd34-4194-848c-f15d2b296ab3/resourceGroups/rg-app-test/providers/Microsoft.Resources/deployments/appdeploy",
        "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-deployment-operations for usage details.",
        "details": [
            {
                "code": "ResourceDeploymentFailure",
                "target": "/subscriptions/0fbef83c-cd34-4194-848c-f15d2b296ab3/resourceGroups/rg-app-test/providers/Microsoft.Resources/deployments/plan2",
                "message": "The resource write operation failed to complete successfully, because it reached terminal provisioning state 'Failed'.",
                "details": [
                    {
                        "code": "DeploymentFailed",
                        "target": "/subscriptions/0fbef83c-cd34-4194-848c-f15d2b296ab3/resourceGroups/rg-app-test/providers/Microsoft.Resources/deployments/plan2",
                        "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-deployment-operations for usage details.",
                        "details": [
                            {
                                "code": "ResourceDeploymentFailure",
                                "target": "/subscriptions/0fbef83c-cd34-4194-848c-f15d2b296ab3/resourceGroups/rg-app-test/providers/Microsoft.Resources/deployments/customer8-4xrn45j6h7wtc",
                                "message": "The resource write operation failed to complete successfully, because it reached terminal provisioning state 'Failed'.",
                                "details": [
                                    {
                                        "code": "DeploymentFailed",
                                        "target": "/subscriptions/0fbef83c-cd34-4194-848c-f15d2b296ab3/resourceGroups/rg-app-test/providers/Microsoft.Resources/deployments/customer8-4xrn45j6h7wtc",
                                        "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-deployment-operations for usage details.",
                                        "details": [
                                            {
                                                "code": "BadRequest",
                                                "target": "/subscriptions/0fbef83c-cd34-4194-848c-f15d2b296ab3/resourceGroups/rg-app-test/providers/Microsoft.Resources/deployments/customer8-4xrn45j6h7wtc",
                                                "message": "{\r\n  \"Code\": \"BadRequest\",\r\n  \"Message\": \"Repository 'GetSiteSourceControl' operation failed with System.Net.WebException: The remote server returned an error: (500) Internal Server Error.\\r\\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\\r\\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.Utilities.WebRequestHelper.<GetResponseAsyncWithTimeout>d__10.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Utilities\\\\WebRequestHelper.cs:line 81\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.TrackerContext.<GetResponseAsync>d__41.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 756\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.<GetSettingAsync>d__32.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 378\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.ExternalSiteRepositoryProvider.<GetSiteSourceControl>d__4.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\ExternalSiteRepositoryProvider.cs:line 35\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.<>c__DisplayClass368_0.<<GetSiteSourceControl>b__0>d.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 11441\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.AsyncHelper.RunSync[TResult](Func`1 func)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.GetSiteSourceControl(String subscriptionName, String webspaceName, String name) in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 0.\",\r\n  \"Target\": null,\r\n  \"Details\": [\r\n    {\r\n      \"Message\": \"Repository 'GetSiteSourceControl' operation failed with System.Net.WebException: The remote server returned an error: (500) Internal Server Error.\\r\\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\\r\\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.Utilities.WebRequestHelper.<GetResponseAsyncWithTimeout>d__10.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Utilities\\\\WebRequestHelper.cs:line 81\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.TrackerContext.<GetResponseAsync>d__41.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 756\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.<GetSettingAsync>d__32.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 378\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.ExternalSiteRepositoryProvider.<GetSiteSourceControl>d__4.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\ExternalSiteRepositoryProvider.cs:line 35\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.<>c__DisplayClass368_0.<<GetSiteSourceControl>b__0>d.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 11441\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.AsyncHelper.RunSync[TResult](Func`1 func)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.GetSiteSourceControl(String subscriptionName, String webspaceName, String name) in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 0.\"\r\n    },\r\n    {\r\n      \"Code\": \"BadRequest\"\r\n    },\r\n    {\r\n      \"ErrorEntity\": {\r\n        \"ExtendedCode\": \"05007\",\r\n        \"MessageTemplate\": \"Repository '{0}' operation failed with {1}.\",\r\n        \"Parameters\": [\r\n          \"GetSiteSourceControl\",\r\n          \"System.Net.WebException: The remote server returned an error: (500) Internal Server Error.\\r\\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\\r\\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.Utilities.WebRequestHelper.<GetResponseAsyncWithTimeout>d__10.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Utilities\\\\WebRequestHelper.cs:line 81\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.TrackerContext.<GetResponseAsync>d__41.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 756\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.<GetSettingAsync>d__32.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 378\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.ExternalSiteRepositoryProvider.<GetSiteSourceControl>d__4.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\ExternalSiteRepositoryProvider.cs:line 35\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.<>c__DisplayClass368_0.<<GetSiteSourceControl>b__0>d.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 11441\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.AsyncHelper.RunSync[TResult](Func`1 func)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.GetSiteSourceControl(String subscriptionName, String webspaceName, String name) in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 0\"\r\n        ],\r\n        \"Code\": \"BadRequest\",\r\n        \"Message\": \"Repository 'GetSiteSourceControl' operation failed with System.Net.WebException: The remote server returned an error: (500) Internal Server Error.\\r\\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\\r\\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.Utilities.WebRequestHelper.<GetResponseAsyncWithTimeout>d__10.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Utilities\\\\WebRequestHelper.cs:line 81\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.TrackerContext.<GetResponseAsync>d__41.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 756\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.SiteRepositoryProvider.<GetSettingAsync>d__32.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\SiteRepositoryProvider.cs:line 378\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.ExternalSiteRepositoryProvider.<GetSiteSourceControl>d__4.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Providers\\\\ExternalSiteRepositoryProvider.cs:line 35\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.<>c__DisplayClass368_0.<<GetSiteSourceControl>b__0>d.MoveNext() in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 11441\\r\\n--- End of stack trace from previous location where exception was thrown ---\\r\\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\\r\\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\\r\\n   at Microsoft.Web.Hosting.AsyncHelper.RunSync[TResult](Func`1 func)\\r\\n   at Microsoft.Web.Hosting.Administration.WebCloudController.GetSiteSourceControl(String subscriptionName, String webspaceName, String name) in C:\\\\__w\\\\1\\\\s\\\\src\\\\Hosting\\\\AdministrationService\\\\Microsoft.Web.Hosting.Administration\\\\Controllers\\\\WebCloudController.cs:line 0.\"\r\n      }\r\n    }\r\n  ],\r\n  \"Innererror\": null\r\n}"
                                            }
                                        ]
                                    }
                                ]
                            }
                        ]
                    }
                ]
            }
        ]
    }
}